src/components/Footer.js

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import "./Footer.css";
+import FooterSnsItem from "./FooterSnsItem";
+import { Link } from "react-router-dom";
+
+const snsList = [
+  {
+    id: 1,
+    name: "facebook",
+    url: "https://www.facebook.com/",
+  },
+  {
+    id: 2,
+    name: "twitter",
+    url: "https://twitter.com/",
+  },
+  {
+    id: 3,
+    name: "youtube",
+    url: "https://www.youtube.com/",
+  },
+  {
+    id: 4,
+    name: "instagram",
+    url: "https://www.instagram.com/",
+  },
+];
+
+export default function Footer() {
+  return (
+    <footer className='footer'>
+      <div className='footer__inner'>
+        <p className='footer__address'>©codeit - 2023</p>
+        <ul className='footer__nav'>
+          <li className='footer__nav-li'>
+            <Link to='/privacy'>Privacy Policy</Link>
+          </li>
+          <li className='footer__nav-li'>
+            <Link to='/faq'>FAQ</Link>
+          </li>
+        </ul>
+        <ul className='footer__sns'>
+          {snsList.map((item) => (
+            <FooterSnsItem
+              key={item.id}
+              id={item.id}
```

Comment:
실제 id 가 있다면 현재 작업하신 것처럼 id 를 활용하는 것이 더 좋고,

혹시 데이터의 구분자가 간단히 필요한 경우라면 index 를 활용할 수도 있습니다! 
(index 를 활용한다면 snsList 작성하실 때 각각 id 를 넣어줄 필요가 없겠네요)

```suggestion
          {snsList.map((item, index) => (
            <FooterSnsItem
              key={item.id}
              index={index}
```

참고로 아이템들의 순서가 바뀔 수 있는 경우에는 index 사용에 주의해야합니다

-> 이런 방식도 있다 참고차 말씀드리는거라 변경하지 않으셔도 됩니다!

Reply:
반복될만한 코드인데 데이터 분리하고 map 사용해서 확장성 좋게 작업하셨네요 👍🏽